app/models/inherited_proofing/mhv_identity_data.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module InheritedProofing
```

Comment:
What do you think about adding specs? I've heard some arguments against adding validator specs (I guess the argument is they're unnecessary), but what I do like is it forces you to create a factorybot object and just sets up the file which helps for specs in the future. 

Refer to this for an example: https://github.com/department-of-veterans-affairs/vets-api/pull/9364/files